### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by gupta-arpan**
## Project

Sunbird-inQuiry

### Changes

Updated Name

### Notes

If there are any further notes.

### CODE REVIEW

File: yarn.lock
Comment (photon0205) [inline yarn.lock:1]: remove this `yarn.lock` file

File: .gitignore
Comment (photon0205) [inline .gitignore:1]: remove yarn.lock from `.gitignore` and keep it unchanged

**Review by photon0205 — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (gupta-arpan) [reply .gitignore:1]: > remove yarn.lock from `.gitignore` and keep it unchanged

Donem kindly check

Comment (photon0205) [reply .gitignore:1]: @gupta-arpan do one thing, download the yarn.lock file from the repo and copy and paste it as it is in the directory and then then commit it in, as i meant that it should not be tracked, not to completely delete it.

Comment (gupta-arpan) [reply .gitignore:1]: > @gupta-arpan do one thing, download the yarn.lock file from the repo and copy and paste it as it is in the directory and then then commit it in, as i meant that it should not be tracked, not to completely delete it.

Done, kindly look

**Review by photon0205 — APPROVED** [review]

**Review by Apoorvgarg-creator — APPROVED** [review]

**Pull request merged**